src/OrchardCore.Modules/OrchardCore.PublishLater/Indexes/PublishLaterPartIndexProvider.cs

```diff
@@ -70,10 +70,10 @@ public void Describe(DescribeContext<ContentItem> context)
 
                 return new PublishLaterPartIndex
                 {
-                    ContentItemId = part.ContentItem.ContentItemId,
```

Comment:
Needed for `part.ScheduledPublishUtc`

Reply:
> Needed for part.ScheduledPublishUtc

Still I'm not sure why the `part.ContentItem` is null, may be something missing in the driver, I need to check

Reply:
Yeah, part.ScheduledPublishUtc is correctly populated.
I think `part.ContentItem` isn't null otherwise there would be a NRE, but all of it's properties are null.

Reply:
Seems the issue it's not populated in the display driver, so I might need to check this, what you did is a workaround